MANUAL_REGRESSION.md

```diff
@@ -488,6 +491,21 @@ Given user opened the link with `?uploadFallback=false` flag
     - "Check readability" text and back arrow retain the colour
     - Back navigation in the browser doesn't cause any other UI changes in the SDK
 
+### 32a. Check happy path flow of live document capture on mobile devices with media recorder API support
+(on private mode of both Android Chrome and iOS Safari mobile browsers)
```

Comment:
We need to look into running the tests on BrowserStack on mobile browsers so that kind of scenarios could be automated too.

Reply:
I agree. Would be good to raise a spike ticket for this. I was going to do this as an automated test, then realised we don't have any automated mobile browser tests yet.

Reply:
Cool. I will create one now. Spike to try to automate this scenario using BrowserStack.